src/services/natsService.ts

```diff
@@ -14,6 +14,7 @@ export class NatsService implements IStartupService {
   functionName = '';
   NatsConn?: NatsConnection;
   logger?: ILoggerService | Console;
+  jc = JSONCodec();
```

Comment:
JSONCodec just does JSON.stringify anyway - so if we accept this, we'll have to revert it for fastjsonify. So happy with using it now, but I think RTK is going to revert. 

`function JSONCodec(reviver) {
    return {
        encode(d) {
            try {
                if (d === undefined) {
                    // @ts-ignore: json will not handle undefined
                    d = null;
                }
                return encoders_1.TE.encode(JSON.stringify(d));`